docker-compose.yaml

```diff
@@ -16,7 +16,7 @@ services:
     restart: on-failure
 
   starknet:
-    image: ghcr.io/dojoengine/dojo:v1.0.0-alpha.2
+    image: ghcr.io/dojoengine/dojo:v0.7.3
```

Comment:
I think that if we want to downgrade properly this, we should downgrade also the other places of the docker/hive stuffs where we have `v1.0.0-alpha.2` in order to keep `v1.0.0-alpha.2` only in the cargo for unit tests and `v0.7.3` everywhere else for the moment, wdyt?